src/algorithms/derivatives.jl

```diff
@@ -28,6 +28,7 @@ Base.:*(h::Union{MPO_∂∂C,MPO_∂∂AC,MPO_∂∂AC2}, v) = h(v);
 (h::MPO_∂∂C)(x) = ∂C(x, h.leftenv, h.rightenv);
 (h::MPO_∂∂AC)(x) = ∂AC(x, h.o, h.leftenv, h.rightenv);
 (h::MPO_∂∂AC2)(x) = ∂AC2(x, h.o1, h.o2, h.leftenv, h.rightenv);
+(h::DerivativeOperator)(v, ::Number) = h(v)
```

Comment:
I am completely fine with this as default behaviour, but it does kind of circumvent the goal of not allowing you to use a time argument when the hamiltonian is not time dependent?

Reply:
yes it does, but the idea is that the user should not need to fiddle around with the derivative operators so it is "safe" (read: not confusing) to allow a time argument that gets ignored. The reason to define this is so that it plays nice with the `integrate` function which expects a two argument function.